Formula/go.rb

```diff
@@ -1,9 +1,9 @@
 class Go < Formula
   desc "Open source programming language to build simple/reliable/efficient software"
   homepage "https://golang.org"
-  url "https://golang.org/dl/go1.16.6.src.tar.gz"
-  mirror "https://fossies.org/linux/misc/go1.16.6.src.tar.gz"
-  sha256 "a3a5d4bc401b51db065e4f93b523347a4d343ae0c0b08a65c3423b05a138037d"
+  url "https://golang.org/dl/go1.17rc1.src.tar.gz"
+  mirror "https://fossies.org/linux/misc/go1.17rc1.src.tar.gz"
```

Comment:
```suggestion
  mirror "https://fossies.org/linux/misc/go1.17rc1.src.tar.gz"
  version "1.17"
```

Reply:
It also needs a whitelist

Reply:
Still stuck at the same issue:

    Stable version URLs should not contain rc1

> It also needs a whitelist

Could you elaborate? Not sure what to do to advance CI beyond the audit check.